src/i18n/it_IT/buildings.ts

```diff
@@ -823,4 +823,136 @@ export default {
         special: '',
         icon: 'plane',
     },
+    25: {
+        caption: 'Ufficio Polizia Locale',
+        color: '#007700',
+        coins: 25,
+        credits: 75_000,
+        levelPrices: {
+            credits: [10_000, 25_000, 50_000, ...Array(8).fill(100_000), 750_000, 875_000],
+            coins: [5, 15, 25, ...Array(8).fill(20)],
+        },
+        extensions: [],
+        levelcost: [],
+        maxBuildings: 'Nessun limite',
+        maxLevel: 11,
+        special: '',
+        startPersonnel: 6,
+        startVehicles: ['Auto della polizia locale'],
+        schoolingTypes: ['Polizia'],
+        schools: [8],
+        startParkingLots: 1,
+        startCells: 0,
+        icon: 'building-shield',
+    },
+    26: {
+        caption: 'Molo Guardia Costiera',
+        color: '#007700',
+        coins: 50,
+        credits: 300_000,
+        levelPrices: {
+            credits: [],
+            coins: [],
+        },
+        extensions: [],
+        levelcost: [],
+        maxBuildings: 'Nessun limite',
+        maxLevel: 11,
+        special: '',
+        startPersonnel: 6,
+        startVehicles: [],
+        schoolingTypes: ['Accademia Navale'],
+        schools: [8],
+        startParkingLots: 1,
+        startCells: 0,
+        icon: 'building-shield',
+    },
+    27: {
+        caption: 'Accademia Navale',
+        color: '#007700',
+        coins: 50,
+        credits: 500_000,
+        levelPrices: {
+            credits: [],
+            coins: [],
+        },
+        extensions: [],
+        levelcost: [],
+        maxBuildings: 'Nessun limite',
+        maxLevel: 0,
+        special: '',
+        startPersonnel: 6,
+        startVehicles: [],
+        schoolingTypes: ['Polizia'],
+        schools: [8],
+        startParkingLots: 1,
+        startCells: 0,
+        icon: 'building-shield',
+    },
```

Comment:
that doesn't seem right, it should be a school, similar to buildings 1, 3 and 8.

Reply:
Solved